tests/functional/spec/list_collector_content.spec.js

```diff
@@ -0,0 +1,129 @@
+import AnyOtherCompaniesOrBranchesPage from "../generated_pages/list_collector_content_page/any-other-companies-or-branches.page.js";
+import AnyCompaniesOrBranchesAddPage from "../generated_pages/list_collector_content_page/any-other-companies-or-branches-add.page.js";
+import AnyCompaniesOrBranchesPage from "../generated_pages/list_collector_content_page/any-companies-or-branches.page";
+import CompaniesSummaryPage from "../generated_pages/list_collector_content_page/section-companies-summary.page";
+import HubPage from "../base_pages/hub.page";
+import ListCollectorFirstRepeatingBlockPage from "../generated_pages/list_collector_content_page/companies-repeating-block-1-repeating-block.page";
+import ListCollectorSecondRepeatingBlockPage from "../generated_pages/list_collector_content_page/companies-repeating-block-2-repeating-block.page";
+import ListCollectorContentPage from "../generated_pages/list_collector_content_page/list-collector-content.page";
+import ListCollectorContentSectionSummaryPage from "../generated_pages/list_collector_content_page/section-list-collector-contents-summary.page";
+
+describe("List Collector Section Summary and Summary Items", () => {
+  describe("Given I launch the test list collector section summary items survey", () => {
+    beforeEach(async () => {
+      await browser.openQuestionnaire("test_list_collector_content_page.json");
+    });
+    it("When I get to the Hub, Then from there the next block in list collector content section should be list collector content page.", async () => {
+      await fillInListCollectorSection();
+      await expect(await browser.getUrl()).to.contain(HubPage.url());
+      await expect(await $(HubPage.summaryRowState("section-list-collector-contents")).getText()).to.contain("Not started");
+      await $(HubPage.submit()).click();
+      await expect(await browser.getUrl()).to.contain(ListCollectorContentPage.url());
+    });
+    it("When I get to the list collector content page, Then the relevant content is displayed.", async () => {
+      await fillInListCollectorSection();
+      await $(HubPage.submit()).click();
+      await expect(await $(await $("body")).getHTML()).to.contain(
```

Comment:
Do we not have selectors for content pages already?

Reply:
There is `definitionContent()` selector but we cannot use it here. I narrowed it by using the id, so `expect(await $("#main-content").getText())`, can possibly narrow it down more by doing `expect(await $("#main-content > div > div > p").getText())` but do we want to?

Reply:
We tackle it separetly.